src/Nova-GPU-Driver.md

```diff
@@ -1,62 +1,92 @@
 # Nova GPU Driver
 
-Nova is a driver for GSP-based Nvidia GPUs that is currently under development
-and is being written in Rust.
-
-Currently, the objective is to upstream Rust abstractions for the relevant
-subsystems a prerequisite for the actual driver. Hence, the first mainline
-version of Nova will be a stub driver which helps establishing the necessary
-infrastructure in other subsystems (notably PCI and DRM).
+Nova is a driver for GSP (global system processor) based Nvidia GPUs. It is
+intended to become the successor of Nouveau as the mainline driver for Nvidia
+(GSP) GPUs on Linux.
 
 ## Contact
 
-To contact the team and / or participate in development, please use the mailing
-list: nouveau@lists.freedesktop.org
+Available communication channels are:
+
+- The mailing list: nouveau@lists.freedesktop.org
+- IRC: #nouveau on OFTC
+- [Zulip Chat](https://rust-for-linux.zulipchat.com/#narrow/channel/509436-Nova)
 
 
 ## Resources
 
-- [Official Source Tree](https://gitlab.freedesktop.org/drm/nova)
-- [Announcement E-Mail](https://lore.kernel.org/dri-devel/Zfsj0_tb-0-tNrJy@cassiopeiae/)
+The parts that are already in mainline Linux can be found in
+`drivers/gpu/nova-core/` and `drivers/gpu/drm/nova/`
+
+The out-of-tree repository where active development takes place is located on
```

Comment:
I know what you mean, but I think calling it out-of-tree is misleading.We could say "development repository for the in-tree driver" instead.